content/docs/use-cases/ci-and-ml.md

```diff
@@ -0,0 +1,172 @@
+# CI and ML
+
+ML engineers and data scientists face many unique challenges. Some of these can
```

Comment:
We can make the first sentence a little more interesting- maybe something along the lines of, "ML engineers and data scientists face many challenges owing to the complexity of the typical project." What do you think?

Reply:
I agree that this sentence can be rephrased in a more appealing way. But the challenges are not due to the "complexity". After all, non-data/ML software projects can be very complex as well.

Maybe: "ML engineers and data scientists face many challenges derived from the dependency on data and complexity of models and other reasons"?

Reply:
That would be better- 

FYI, I mean complexity not as in cyclomatic or algorithmic complexity, but as in system complexity (see http://papers.nips.cc/paper/5656-hidden-technical-debt-in-machine-learning-systems.pdf). Technical debt due to hardware, data dependencies, feedback loops, etc.